src/js/post-911-gib-status/components/EnrollmentPeriod.jsx

```diff
@@ -1,35 +1,95 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Scroll from 'react-scroll';
 
 import InfoPair from './InfoPair';
 
+import { getScrollOptions } from '../../common/utils/helpers';
 import { formatDateShort } from '../utils/helpers';
 
+const scroller = Scroll.scroller;
+
 class EnrollmentPeriod extends React.Component {
+  constructor() {
+    super();
+    this.scrollToEnrollment = this.scrollToEnrollment.bind(this);
+    this.toggleHistory = this.toggleHistory.bind(this);
+    this.state = { historyExpanded: false };
+  }
+
+  scrollToEnrollment() {
+    const options = getScrollOptions({ delay: 2 });
+    scroller.scrollTo(`collapsible-${this.props.id}`, options);
+  }
+
+  toggleHistory() {
+    const shouldExpand = !this.state.historyExpanded;
+    this.setState({ historyExpanded: !this.state.historyExpanded });
+    if (shouldExpand) {
+      this.scrollToEnrollment();
+    }
+  }
+
   render() {
-    const { enrollment } = this.props;
+    const { enrollment, id } = this.props;
+    const amendments = enrollment.amendments || [];
+
+    // TODO: should fullTimeHours be used anywhere?
+    // const fullTimeHours = <InfoPair label="Full-time Hours" value={enrollment.fullTimeHours}/>;
+    const onCampusHours = <InfoPair label="On-campus Hours" value={enrollment.onCampusHours}/>;
+    const onlineHours = <InfoPair label="Online Hours" value={enrollment.onlineHours}/>;
 
-    const fullTimeHours = <InfoPair label="On-campus Hours" value={enrollment.fullTimeHours}/>;
-    const distanceHours = <InfoPair label="Distance Hours" value={enrollment.distanceHours}/>;
-    const typeOfChange = enrollment.amendmentList && enrollment.amendmentList.length > 0 &&
-      <InfoPair label="Type of Change" value={enrollment.amendmentList[0].type}/>;
-    const changeEffectiveDate = enrollment.amendmentList && enrollment.amendmentList.length > 0 &&
-      <InfoPair label="Change Effective Date" value={enrollment.amendmentList[0].changeEffectiveDate}/>;
+    const amendmentList = this.state.historyExpanded ? (
+      <div id={`collapsible-${id}`} className="usa-accordion-content">
+        {amendments.map((amendment, index) => {
+          return (
+            <div key={`amendment-${this.props.key}-${index}`}>
```

Comment:
Oops, this is no longer needed. I had originally thought each key needed to be unique across the whole document, but it only needs to be unique among its siblings, so this will just be `amendment-${index}`